crates/ide_db/src/helpers/insert_use.rs

```diff
@@ -189,6 +189,9 @@ pub fn try_merge_imports(
     if !eq_visibility(lhs.visibility(), rhs.visibility()) {
         return None;
     }
+    if !lhs.attrs().eq(rhs.attrs()) {
```

Comment:
```suggestion
    if lhs.attrs() != rhs.attrs() {
```

Reply:
This actually uses `Iterator.eq`, as there is no `impl PartialEq for AstChildren<syntax::ast::Attr>`. It should probably be `Iterator.ne` though

Reply:
`attrs` returns an `Iterator` here hence the `eq` call I'm pretty sure. But this will always evaluate to false(true after the negation), due to `AstNode`s only being equal when they point to the same text If I remember correctly. This should be changed to check for textual equality similar to how `eq_visibility` works for now.

Reply:
I changed it to compare the attrs textually, as in eq_visibility

Reply:
Seems good to me 👍